migration/migrations/course/20180723124511_regrade_per_assignment.py

```diff
@@ -0,0 +1,10 @@
+def up(config, conn, semester, course):
+    with conn.cursor() as cursor:
+        cursor.execute("ALTER TABLE electronic_gradeable DROP COLUMN if exists eg_regrade_request_date")
```

Comment:
I don't think we want to drop the column.  If you read the migrations doc, the goal is to preserve data.